Resources/Prototypes/Entities/Constructible/Power/cloning_machine.yml

```diff
@@ -0,0 +1,43 @@
+﻿- type: entity
+  id: CloningPod
+  name: Cloning Pod
+  description: A Cloning Pod. 50% reliable.
+  components:
+    - type: Sprite
+      netsync: false
+      sprite: Objects/Specific/Medical/cloning.rsi
+      layers:
+        - state: pod_0
+          map: ["enum.CloningPodVisualLayers.Machine"]
+    - type: PowerReceiver
+    - type: Icon
+      sprite: Objects/Specific/Medical/cloning.rsi
+      state: pod_0
+    - type: Anchorable
+    - type: Clickable
+    - type: InteractionOutline
+    - type: Collidable
+      shapes:
+        - !type:PhysShapeAabb
+          bounds: "-0.5,-0.25,0.5,0.25"
+          layer:
+            - Opaque
+            - Impassable
+            - MobImpassable
+            - VaultImpassable
+      IsScrapingFloor: true
+    - type: Physics
```

Comment:
i think Physics is replaced by Collidable, so you could just move the `anchored` up there?

Reply:
Tried it, however the cloning pod was still pushable.